lib/matplotlib/tests/test_axes.py

```diff
@@ -7604,4 +7613,4 @@ def test_bezier_autoscale():
     ax.autoscale()
     # Bottom ylim should be at the edge of the curve (-0.5), and not include
     # the control point (at -1)
-    assert ax.get_ylim()[0] == -0.5
+    assert ax.get_ylim()[0] == -0.5
```

Comment:
Why did this line change?

Reply:
I don't see a difference between these two lines

Reply:
It is missing a newline after it. Some editors annoyingly will strip off a trailing newline at the end of a file when saving.

Reply:
fixed